### PR TITLE
Create and use AP_CAMERA_ENABLED

### PR DESCRIPTION
**Pull request by peterbarker**
Builds tested locally by using an extra.hwdef

build_options.py yet to be tested

### CODE REVIEW

**Comment by peterbarker** [comment]
Saves 4kB when disabled

**Comment by tatsuy** [comment]
you can also remove the define from Blimp/config.h.
https://github.com/ArduPilot/ardupilot/pull/20020/commits/3ca9ce864bc72090648c2a52e29442d1d306c6d8

**Comment by peterbarker** [comment]
> you can also remove the define from Blimp/config.h like [this](https://github.com/ArduPilot/ardupilot/pull/20020/commits/3ca9ce864bc72090648c2a52e29442d1d306c6d8).

Thanks, I've cherry-picked that in here.

**Comment by peterbarker** [comment]
I've tested the custom build server with this.

There's no difference in build size for CubeOrange master vs this PR.

File: libraries/GCS_MAVLink/GCS_Common.cpp [hunk 1]
```diff
@@ -4415,10 +4419,11 @@ MAV_RESULT GCS_MAVLINK::handle_command_long_packet(const mavlink_command_long_t
 
     case MAV_CMD_DO_DIGICAM_CONFIGURE:
     case MAV_CMD_DO_DIGICAM_CONTROL:
+#if AP_CAMERA_ENABLED
```
Comment (rmackay9) [inline libraries/GCS_MAVLink/GCS_Common.cpp:4422]: little fall through issue.

**Comment by peterbarker** [comment]
Labelled as MergeOnCIPass; this went through a DevCall a long time back and was OK once it was tested on hw, which I've just done.

**Pull request merged**